Library/Util.ts

```diff
@@ -178,5 +179,18 @@ class Util {
 
         return true;
     }
+
+    public static getCorrelationContextTarget(response: http.ClientResponse | http.ServerRequest) {
+        const contextHeaders = response.headers && response.headers[RequestResponseHeaders.requestContextHeader];
+        if (contextHeaders) {
+            const keyValues = contextHeaders.split(",");
+            for(let i = 0; i < keyValues.length; ++i) {
+                const keyValue = keyValues[i].split("=");
+                if (keyValue.length == 2 && keyValue[0] == RequestResponseHeaders.requestContextTargetKey) {
```

Comment:
In theory shouldn't you be using TargetKey for clientResponse and SourceKey for Server request. I know both values are the same so it doesn't really matter, but conceptually you have defined two different constants.